rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/LocalDataTransferServer.java

```diff
@@ -217,52 +217,62 @@ public void stop()
     public void run()
     {
       try {
-        log.debug("Connected to node - " + localDataTransferClient.getRemoteAddress());
-        ByteBuffer dataInfo = ByteBuffer.allocate(CacheConfig.getMaxHeaderSize(conf));
+        log.debug("Connected to node - " + localDataTransferClient.getLocalAddress());
```

Comment:
We know the remote address as it will be <remote node>:8898, but to debug issues we may need to see the local ephemeral port. That's why I changed it. Or, I will keep both of them, we never know which one we may need to debug.